src/multivariate/mvnormal.jl

```diff
@@ -236,7 +236,7 @@ Base.show(io::IO, d::MvNormal) =
 ### Basic statistics
 
 length(d::MvNormal) = length(d.μ)
-mean(d::MvNormal) = convert(Vector, d.μ)
+mean(d::MvNormal) = copy(convert(Vector, d.μ))
```

Comment:
This should be enough (avoiding one copy when the object isn't a `Vector` already):
```suggestion
mean(d::MvNormal) = Vector(d.μ)
```

Reply:
That still copies:
```
julia> x = [1,2,3];

julia> y = Vector(x);

julia> y === x
false
```